Assets/Scripts/EnemyVision.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemyVision : MonoBehaviour {
+    public float viewDist = 5.0f;
+    public bool sight = false;
+
+    public float projectileWait = 1;
+    public float ProjectileSpeed = 1;
+    public GameObject ProjectilePrefab;
+
+    private RaycastHit _hit;
+    private Vector3 _playerLocation, _raycastDirection, _startVec;
+    private GameObject _player;
+
+	// Use this for initialization
+	void Start () {
+        _player = GameObject.FindGameObjectWithTag("Player");
+        StartCoroutine(EnemyShoot());
```

Comment:
We should only need to call EnemyShoot() at the time we need it. Having a coroutine run all the time adds up and slows stuff down later, especially with multiple enemies.